src/resources/Readings.js

```diff
@@ -1,16 +1,73 @@
-import React from "react";
+import PropTypes from "prop-types";
+import { useState } from "react";
 import "./resources.css";
 
+const fakeData = require("./fakeData");
+
+// tags is currently unused
+function ToggleItem({ id, title, publicationDate, description, url, tags }) {
+	const [showButton, setShowButton] = useState(false);
+
+	return (
+		<li className="resource-list-item" key={id}>
+			<button
+				type="button"
+				className={`resource-list-title ${
+					showButton && "resource-list-title-active"
+				}`}
+				onClick={() => setShowButton(prevState => !prevState)}
+			>
+				{title}
+			</button>
+
+			<li
+				className={`resource-list-description ${
+					showButton && "resource-list-description-active"
+				}`}
+			>
+				{description}
+				{"\n\n"}
+				<a
+					className="resource-link"
+					href={url}
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					Read it here. {/* Open link in new tab*/}
+				</a>
+				{"\n\n"}
+				Publication date: {publicationDate}
+				{"\n"}
+				Tags: {tags.join(", ")}
```

Comment:
It's ok to add  {"\n"} to give more space between lines, but I don't normally do it. Instead, I do `<p>{description}</p>`. I will modify `<p>` in css like `.resource-list-description p { color: #fff }`. 

It will modify p tag specifically inside `.resource-list-description`. Or, you can add `id` to `<p>`. Both your and my method yield same result, but my method can be reusable for many files with the same pattern. 

It's up to you to use which method

Reply:
I used your method! Thanks!